app/controllers/home_controller.rb

```diff
@@ -1,6 +1,6 @@
 class HomeController < ApplicationController
   def welcome
-    @card = Card.random
+    @card = Card.random(current_user.id)
```

Comment:
@davetoxa that won't be able to retrieve random cards from the due_today array though